evap/student/templates/student_vote.html

```diff
@@ -84,6 +84,23 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
             </div>
         {% endif %}
 
+        {% if show_dropout_questionnaire %}
```

Comment:
Can't we just do
```suggestion
        {% if evaluation_form_group_dropout %}
```
and remove this variable entirely? Or is there a chance that dropout is enabled, students want to drop out, but we don't assign any dropout questionnaires? No right?